data-prepper-plugins/csv-processor/src/main/java/org/opensearch/dataprepper/plugins/codec/csv/CsvInputCodec.java

```diff
@@ -61,22 +62,12 @@ private void parseBufferedReader(final BufferedReader reader, final Consumer<Rec
         }
 
         MappingIterator<Map<String, String>> parsingIterator = mapper.readerFor(Map.class).with(schema).readValues(reader);
-        boolean hasNextValue;
```

Comment:
@umayr-codes , You have reverted some changes in your most recent rebase. See this commit: https://github.com/opensearch-project/data-prepper/commit/bc75494167d6befe3778493749678a501afbe1a9

I believe you can resolve this issue by changing all the code from this line through the end of this file back. Basically, we should not see any diffs after line 41.

You may also need to check the imports above.

Reply:
Its done David!